pages/Identity/Registry.md

```diff
@@ -161,14 +171,15 @@ interface IRegistry {
      * @param v EIP-155 calculated Signature v value
      * @param newAddr New address for the DSNP Id to point at
      * @param handle The handle to modify
+     * @param nonce The expected nonce for this update (must match return value of resolveHandleToNonce())
      * 
      * MUST be signed by someone who is authorized on the contract
      *      via `IDelegation(oldAddr).isAuthorizedTo(ecrecovedAddr, Permission.OWNERSHIP_TRANSFER, block.number)`
      * MUST check that newAddr implements IDelegation interface
      * TODO: FIX THE ISSUE OF newAddr not being a part of the creation
      * MUST emit DSNPRegistryUpdate
      */
-    function changeAddressByEIP712Sig(bytes32 r, bytes32 s, uint32 v, address newAddr, string handle) external;
+    function changeAddressByEIP712Sig(bytes32 r, bytes32 s, uint32 v, address newAddr, string handle, uint64 nonce) external;
```

Comment:
@acruikshank Are we sure we want a 64 bit nonce? (address + id + nonce) 160+64+64 > 256 So our struct would take more than one storage slot. A 32 bit nonce would fit us into a single 256 slot 160+64+32.

Reply:
I'll change it. I thought this would be handled by the number encoding when the struct is packed, but I can't find a definitive answer on that.